src/pint/scripts/pintk.py

```diff
@@ -77,6 +79,34 @@ def initUI(self):
         self.helpMenu.add_command(label="Plk Help", command=lambda: print(helpstring))
         self.menuBar.add_cascade(label="Help", menu=self.helpMenu)
 
+        self.logMenu = tk.Menu(self.menuBar)
+        self.logMenu.add_checkbutton(
+            label="DEBUG",
+            command=lambda: self.changeLogLevel("DEBUG"),
+            variable=self.log_levels["DEBUG"],
+        )
+        self.logMenu.add_checkbutton(
+            label="INFO",
+            command=lambda: self.changeLogLevel("INFO"),
+            variable=self.log_levels["INFO"],
+        )
+        self.logMenu.add_checkbutton(
+            label="WARNING",
+            command=lambda: self.changeLogLevel("WARNING"),
+            variable=self.log_levels["WARNING"],
+        )
+        self.logMenu.add_checkbutton(
+            label="ERROR",
+            command=lambda: self.changeLogLevel("ERROR"),
+            variable=self.log_levels["ERROR"],
+        )
+        self.logMenu.add_checkbutton(
+            label="NOTSET",
```

Comment:
What does NOTSET do? If that's all the option says, the user may be puzzled. Does this provide a way for the user to log nothing at all?

Reply:
I will need to check back on this, I thought one thing when testing it out last night, but it may essentially be a no-op and better left off the list.